packages/react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -817,7 +826,10 @@ const DatePicker = React.forwardRef(function DatePicker(
 DatePicker.propTypes = {
   /**
    * flatpickr prop passthrough. Allows the user to enter a date directly
-   * into the input field
+   * into the input field, Note: If we set this flag to false and a user picks a date, 
+   * they won't be able to remove it by just clicking on it. 
+   * Instead, you'll need to set the value to an empty string yourself if you want to clear the date. 
+   * This will make the input behave like a controlled form element.
```

Comment:
I few like the explanation is too much to address for the user. Might be good to shorten a little bit.

Reply:
Sure will update this one

Reply:
@guidari comment updated, let me know if there are any other changes?